services/content/service.go

```diff
@@ -88,6 +90,19 @@ func (s *Service) Update(ctx context.Context, req *api.UpdateRequest) (*api.Upda
 		return nil, grpc.Errorf(codes.InvalidArgument, "%q failed validation", req.Info.Digest)
 	}
 
+	if req.UpdateMask != nil && len(req.UpdateMask.Paths) > 0 {
```

Comment:
Also, this should be in the content store, as well. We already have a spot where labels are applied. ;)